### ISSUE TITLE
Add preview link for PRs that change docs (using vitepress)

### ISSUE THREAD

**Issue by nedsalk** [FuelLabs/fuels-ts#2455]

**Comment by msensys** [comment]
Hey @nedsalk, what do you think about using `Github pages` for this? Maybe we could use `Stackblitz` for it too.

**Comment by arboleya** [comment]
Let's see how to get that up and running.

**Comment by arboleya** [comment]
I never used the `comments` feature, so I can't say much.

It seems magical, but would that play nicely with GH reviews?

The branch name in the URLs seems the way to go!

**Comment by petertonysmith94** [comment]
> I never used the `comments` feature, so I can't say much.
> 
> It seems magical, but would that play nicely with GH reviews?

Neither have I if I'm honest, maybe we can leave that for now.
I'll go with the branch names - thanks for the input.

**Comment by petertonysmith94** [comment]
I'm going to propose removing our GitHub pages deployment from the release and using solely Vercel.

What are people thoughts/concerns?

**Comment by Dhaiwat10** [comment]
Yeah no need to have it after setting up Vercel @petertonysmith94
